gala/potential/potential/builtin/core.py

```diff
@@ -407,6 +407,23 @@ class BurkertPotential(CPotentialBase):
 
     Wrapper = BurkertWrapper
 
+    
+    @staticmethod
+    def from_r0(r0, units=None):
+        r"""
+        from_r0(r0, units=None)
+
+        Initialize a Burkert potential from the core radius, ``r0``.
+        See Equations 4 and 5 of Mori & Burkert.
+
+        Parameters
+        ----------
+        r0 : :class:`~astropy.units.Quantity`, numeric [length]
+            The core radius of the Burkert potential.
+        """
+        rho_d0 = 1.46e-24 * (r0 / (3.07 * u.kpc))**(-2/3) * (u.g / u.cm**3)
+        return BurkertPotential(rho=rho_d0, r0=r0, units=units)
```

Comment:
```suggestion
    @classmethod
    def from_r0(cls, r0, units=None):
        r"""
        from_r0(r0, units=None)

        Initialize a Burkert potential from the core radius, ``r0``.
        See Equations 4 and 5 of Mori & Burkert.

        Parameters
        ----------
        r0 : :class:`~astropy.units.Quantity`, numeric [length]
            The core radius of the Burkert potential.
        """
        rho_d0 = 1.46e-24 * (r0 / (3.07 * u.kpc))**(-2/3) * (u.g / u.cm**3)
        return cls(rho=rho_d0, r0=r0, units=units)
```
I would make this a class method. Also, can the small constant out front (1.46e-24) be converted to a more common unit system where its value is closer to one? For example, in Msun/pc**3 it is ~0.022.

Reply:
Normally I would wait for you to do the edits, but I'm trying to cut a release today and I want this to be in, so I'll just make the changes here :).